<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherEffectsWidget</class>
 <widget class="QWidget" name="WeatherEffectsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change item</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioTypeNo">
          <property name="text">
           <string>No Weather</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupType_arg0</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTypeRain">
          <property name="text">
           <string>Rain</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupType_arg0</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTypeSnow">
          <property name="text">
           <string>Snow</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupType_arg0</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTypeFog">
          <property name="text">
           <string>Fog</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupType_arg0</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTypeSand">
          <property name="text">
           <string>Sandstorm</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupType_arg0</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Power</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioPowerWeak">
          <property name="text">
           <string>Weak</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupStrength_arg1</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPowerMedium">
          <property name="text">
           <string>Medium</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupStrength_arg1</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPowerStrong">
          <property name="text">
           <string>Strong</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">groupStrength_arg1</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>helpRequested()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="groupType_arg0"/>
  <buttongroup name="groupStrength_arg1"/>
 </buttongroups>
</ui>
